panel/template/base.py

```diff
@@ -402,7 +401,7 @@ def resolve_resources(
                     continue
                 elif scls._css is None:
                     break
-                tmpl_css = scls._css if isinstance(scls._css, list) else [scls._css]
+                tmpl_css = scls._css
```

Comment:
This is a breaking change; this breaks custom templates that use strings for `_css`.